cmd/snap/cmd_snap_op.go

```diff
@@ -665,7 +665,11 @@ func (x *cmdRefresh) showRefreshTimes() error {
 		return err
 	}
 
-	fmt.Fprintf(Stdout, "schedule: %s\n", sysinfo.Refresh.Schedule)
+	if sysinfo.Refresh.Schedule != "" {
```

Comment:
(nitpick^2) I wonder if this should be the other way around (even though it does not make a practical difference I suppose). I.e. to make clear that `sysinfo.Refresh.Timer` is the thing that always wins if it is set?

We should also probably show an (internal) error if no refresh schedule is set at all (and add a test for this).